cmake/AddBipedalLocomotionLibrary.cmake

```diff
@@ -108,6 +109,21 @@ function(add_bipedal_locomotion_library)
   endforeach()
   set_property(GLOBAL PROPERTY umbrella_includes "${umbrella_includes_list}")
 
+  if (NOT ${skip_install_check})
+    foreach(header ${public_headers})
+      get_filename_component(extension ${header} LAST_EXT)
+      if ((extension STREQUAL ".h") OR (extension STREQUAL ".hpp"))
```

Comment:
might as well check for `.tpp`?

Reply:
Usually ``.tpp`` files are included using a "relative" include. See for example https://github.com/dic-iit/bipedal-locomotion-framework/blob/bb27200a62f3d84c779f9d509d0c72d5316505d7/src/YarpUtilities/include/BipedalLocomotion/YarpUtilities/Helper.h#L158

Reply:
Then we fix those in `System` library to use "relative" includes

https://github.com/dic-iit/bipedal-locomotion-framework/blob/bb27200a62f3d84c779f9d509d0c72d5316505d7/src/System/include/BipedalLocomotion/System/FixedStepIntegrator.h#L67

Reply:
Uh, nice catch. Ok I will remove the check on the extension, so that also ``tpp`` files are included.

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-framework/pull/247/commits/8aef41efb03698403ff7066204cb74ff113fbeec

Reply:
Great!